.editorconfig

```diff
@@ -3,7 +3,7 @@ root = true
 
 [*]
 indent_style = space
```

Comment:
I the issue is that editorconfig is telling some editors to use spaces, when the style rules (enforced by JSHint/JSLint) use tabs.